internal/envconfig/envconfig.go

```diff
@@ -55,6 +55,8 @@ var (
 	// setting the environment variable "GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST"
 	// to "true".
 	NewPickFirstEnabled = boolFromEnv("GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST", false)
+	//HTTPProxy is set to empty string if HTTPS_PROXY env it is not set.
```

Comment:
space before HTTPProxy

Reply:
Also, name should be HTTPSProxy as well?

Reply:
Yes, it should be HTTPSProxy, changed.